setup.py

```diff
@@ -70,7 +70,7 @@ def build(cls):
         try:
             import sass
         except ModuleNotFoundError:
-            pip.main(['install', 'libsass'])
+            check_call([sys.executable, '-m', 'pip', 'install', 'libsass'])
```

Comment:
`pip install -r requirements.txt`. In that way we dont need to keep here hardcoded the libraries, right ? `libasass` or any kind of dependency here on source code is not good.